components/SearchBar.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useRef } from "react";
+
+export default function SearchBar({ value, onChange }) {
```

Comment:
I think this should be called `SearchInput`

Reply:
Done